ray_data_eval/simulator/benchmark.py

```diff
@@ -0,0 +1,45 @@
+import logging
+from ray_data_eval.common.pipeline import problems
+from ray_data_eval.simulator.environment import ExecutionEnvironment
+from ray_data_eval.simulator.policies import (  # noqa F401
+    GreedySchedulingPolicy,
+    GreedyWithBufferSchedulingPolicy,
+    GreedyAndAnticipatingSchedulingPolicy,
+    SchedulingPolicy,
+    RatesEqualizingSchedulingPolicy,
+)
+
+logging.disable(logging.CRITICAL)
```

Comment:
uh, why?

Reply:
I am thinking about keeping the output clean:
```
Problem: test_problem
GreedySchedulingPolicy Finished 6
GreedyWithBufferSchedulingPolicy Finished 6
GreedyAndAnticipatingSchedulingPolicy Finished 6
RatesEqualizingSchedulingPolicy Finished 6
---
Problem: multi_stage_problem
GreedySchedulingPolicy Finished 12
GreedyWithBufferSchedulingPolicy Finished 12
GreedyAndAnticipatingSchedulingPolicy Finished 12
RatesEqualizingSchedulingPolicy Finished 11
---
Problem: producer_consumer_problem
GreedySchedulingPolicy Finished 12
GreedyWithBufferSchedulingPolicy Finished 12
GreedyAndAnticipatingSchedulingPolicy Finished 12
RatesEqualizingSchedulingPolicy Finished 12
---
Problem: long_problem
GreedySchedulingPolicy Finished 60
GreedyWithBufferSchedulingPolicy Finished 60
GreedyAndAnticipatingSchedulingPolicy Finished 60
RatesEqualizingSchedulingPolicy Finished 59
---
```

Reply:
removed 
`logging.disable(logging.CRITICAL)`